dio/lib/browser.dart

```diff
@@ -1,2 +1 @@
-export 'src/adapters/browser_adapter.dart' show BrowserHttpClientAdapter;
```

Comment:
Do we need to re-export the new adapter here, ideally with the same name? Otherwise we have a breaking change.